js/flatbuffers.js

```diff
@@ -828,6 +846,14 @@ flatbuffers.ByteBuffer.allocate = function(byte_size) {
   return new flatbuffers.ByteBuffer(new Uint8Array(byte_size));
 };
 
+flatbuffers.ByteBuffer.prototype.clear = function() {
+  var len = this.bytes_.length;
+  for (var i = 0; i < len; i++) {
+    this.bytes_[i] = 0;
```

Comment:
clearing these bytes seems expensive, why not just leave them as-is?

Reply:
I guess I wasn't scientific about adding that, thought it was necessary to reuse the ByteBuffer, but I can see now it isn't.  Removed.